mtags/src/main/scala-2/scala/meta/internal/pc/PcSemanticTokensProvider.scala

```diff
@@ -24,10 +24,20 @@ final class PcSemanticTokensProvider(
         symbol: Option[compiler.Symbol]
     ): Option[Node] = {
       val sym = symbol.fold(tree.symbol)(s => s)
+
+      val isDeclaration = tree match {
+        case df: compiler.ValOrDefDef =>
+          df.rhs.isEmpty
+        case _ => false
+      }
+      val isDefinition = tree match {
+        case _: compiler.DefTree => !isDeclaration
+        case _ => false
+      }
       if (
         !pos.isDefined || sym == null || sym == compiler.NoSymbol || sym.isConstructor
       ) None
-      else Some(makeNode(sym, adjust(pos)._1))
+      else Some(makeNode(sym, adjust(pos)._1, isDefinition, isDeclaration))
```

Comment:
`adjust(pos)._1`, 2 boolean flags. I'd think about using named parameters for `makeNode` method

Reply:
Changed!